okhttp/pom.xml

```diff
@@ -35,6 +35,9 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-javadoc-plugin</artifactId>
                 <version>3.0.0-M1</version>
```

Comment:
Replace with 3.1.0

Reply:
Ah I had done that and didn't push it I guess